src/components/implementation/tests/unit_pmu/unit_pmu.c

```diff
@@ -0,0 +1,26 @@
+#include <cos_kernel_api.h>
+#include <cos_types.h>
+#include <hw_perf.h>
+
+#define NUM_PAGES 12
+
+word_t buf[NUM_PAGES*PAGE_SIZE];
```

Comment:
shouldn't this be u8_t or some such? This is 8 times higher than your constants specify.

I didn't mention this before, but because it keeps happening: please put a space around binary operators.